sage_code/strong_uniform_bounds.py

```diff
@@ -109,20 +123,71 @@ def bound_from_split_type(split_type, eps, q):
         if pil_int == 0:
             zero_detection_flag = True
         else:
-            running_lcm = lcm(pil_int, running_lcm)
+            pil_int = gcd(known_mult_bound, pil_int)
+            running_lcm = gcd(known_mult_bound, lcm(pil_int, running_lcm))
```

Comment:
```suggestion
            running_lcm = lcm(pil_int, running_lcm)
```
Since at this point `pil_int` and `running_lcm` already are divisors of `known_mult_bound` the second gcd is not necessary

Reply:
next commit (not committed yet) will resolve this